templates/confidential-computing/index.html

```diff
@@ -0,0 +1,287 @@
+                width="22",
+                hi_def=True,
+                loading="lazy" ) | safe
+              }}
+            </p>
+          </div>
+          <div class="col-4 col-medium-3 col-small-2">
+            <ul class="p-list--divided u-no-margin--bottom">
+              <li class="p-list__item">
+                <a
+                  href="https://azuremarketplace.microsoft.com/en-us/marketplace/apps/canonical.0001-com-ubuntu-confidential-vm-focal?tab=Overview">AMD
+                  SEV SNP with Ubuntu 20.04 LTS on Azure</a>
+              </li>
+              <li class="p-list__item">
+                <a
+                  href="https://azuremarketplace.microsoft.com/en-us/marketplace/apps/canonical.0001-com-ubuntu-confidential-vm-jammy">AMD
+                  SEV SNP with Ubuntu 22.04 LTS on Azure</a>
+              </li>
+              <li class="p-list__item">
+                <a
+                  href="https://forms.office.com/pages/responsepage.aspx?id=v4j5cvGGr0GRqy180BHbR9_qJVF-zyZCqvwR_iaG2EFUME1BT1hJN0I1VjhGWE9QNUJGREQ5QUlMNS4u">Intel
+                  TDX in private preview with 22.04 LTS on Azure</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <hr />
+        <div class="row">
+          <div class="col-2 col-medium-3 col-small-2 u-pad-logo">
+            <p class="u-no-margin--bottom">
+              {{ image (
+                url="https://assets.ubuntu.com/v1/2c479064-cloud-logo%201.svg",
+                alt="Google Cloud logo",
+                height="22",
+                width="129",
+                hi_def=True,
+                loading="lazy" ) | safe
+              }}
+            </p>
+          </div>
+          <div class="col-4 col-medium-3 col-small-2">
+            <p><a href="https://ubuntu.com/blog/start-your-snp-vms-on-google-cloud">AMD SEV SNP with Ubuntu 22.04 LTS on Google Cloud</a></p>
+          </div>
+        </div>
+        <hr />
+        <div class="row">
+          <div class="col-2 col-medium-3 col-small-2 u-pad-logo">
+            <p class="u-no-margin--bottom">
+              {{ image (
+                url="https://assets.ubuntu.com/v1/8e5cc412-AWS.svg",
+                alt="AWS logo",
+                height="22",
+                width="40",
+                hi_def=True,
+                loading="lazy" ) | safe
+              }}
+            </p>
+          </div>
+          <div class="col-4 col-medium-3 col-small-2">
+            <p><a href="https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/snp-requirements.html">AMD SEV SNP with Ubuntu 23.10 on AWS</a></p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <hr class="is-fixed-width" />
+
+  <section class="p-strip is-deep">
+    <div class="row">
+      <div class="col-12">
+        <p class="p-heading--2">
+          <a href="https://ubuntu.com/engage/introduction-confidential-computing">Download our guide to confidential computing ›</a>
```

Comment:
```suggestion
          <a href="/engage/introduction-confidential-computing">Download our guide to confidential computing ›</a>
```
For internal links we don't need to add the base path